proposals/CloudFileHandle.md

```diff
@@ -30,10 +30,10 @@ if(cloudHandles.length === 0) {
   // File is not synced by any CSP
 }
 for(const cloudHandle of cloudHandles) {
-  if(cloudHandle.provider === 'drive.google.com') {
```

Comment:
it normally never follows the `provider_name` pattern. 
everywhere else it's named camelCase

so maybe should be `providerName`

Reply:
good catch, thanks!